src/core/evaluator.js

```diff
@@ -3143,7 +3143,7 @@ class PartialEvaluator {
       // Heuristic: we have to check if the font is a standard one also
       if (isSymbolicFont) {
         encoding = MacRomanEncoding;
-        if (!properties.file) {
+        if (!properties.file || properties.isInternalFont) {
```

Comment:
Note that I briefly considered just removing this `if`, and instead run its code unconditionally, but decided against it in order to reduce the risk of any future regressions.